ev3/__init__.py

```diff
@@ -1 +1,5 @@
-__all__ = ['ev3dev', 'lego','mindsensors']
```

Comment:
It seems only a newline was added? There is still a space missing. Or do you mean the whole pull?